packages/react-ui-core/src/Counter/Counter.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+export default class Counter extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: this.props.count,
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+  handleClick(count) {
+    this.props.handleClick(count)
+    this.setState({
+      count,
+    })
+  }
+  render() {
+    const { count } = this.state
+    const { theme, label, leftUnit, rightUnit, id } = this.props
+    return (
+      <div id={id} className={theme.container}>
+        <div className={theme.label}>
+          {label}
+        </div>
+        <div className={theme.componentContainer}>
+          <span
+            onClick={() => this.handleClick(this.state.count + 1)}
+            className={classNames('leftUnit', theme.contentInnards, theme.shape)}
+          >
+            {this.props.leftOperator}
+          </span>
+          <span
+            className={classNames(theme.contentInnards, theme.count)}
+          >{`${leftUnit} ${count} ${rightUnit}`}</span>
+          <span
+            onClick={() => this.handleClick(this.state.count - 1)}
+            className={classNames('rightUnit', theme.contentInnards, theme.shape)}
+          >
+            {this.props.rightOperator}
+          </span>
+        </div>
+      </div>
```

Comment:
The theme classnames here don't follow our CEM (component-element-modifier) convention.  I would expect the classnames to look more like:
```
.Counter {} // component
.Counter-active // component-modifier
.Counter_label {} // component-element
.Counter_label-active {} // component-element-modifier
```